tests/fgpyo/util/test_itertools.py

```diff
@@ -0,0 +1,24 @@
+from itertools import product
+from typing import List
+from typing import Tuple
+
+from typing_extensions import TypeAlias
+
+from fgpyo.util.itertools import sorted_product
+
+
+def test_sorted_product() -> None:
+    """sorted_product should return a sorted cartesian product of the input lists."""
+    xs: List[int] = [25, 10, 10, 1]
+    ys: List[int] = [1, 2, 10]
```

Comment:
```suggestion
    ys: List[int] = [10, 2, 1]
```

Reply:
Doh

leaving myself a reminder to verify the inputs are sorted (and in the correct order)